docker/data-science/notebooks/ch10/4.train-and-evaluate-the-model.ipynb

```diff
@@ -510,43 +483,41 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 221,
+   "execution_count": 17,
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/plain": [
-       "[{'id': '570724', 'title': 'The Story of Harry Potter', 'score': 2.4261165},\n",
-       " {'id': '116972',\n",
-       "  'title': 'Discovering the Real World of Harry Potter',\n",
-       "  'score': 2.2478466},\n",
-       " {'id': '672',\n",
-       "  'title': 'Harry Potter and the Chamber of Secrets',\n",
-       "  'score': 2.0174997},\n",
-       " {'id': '671',\n",
-       "  'title': \"Harry Potter and the Philosopher's Stone\",\n",
-       "  'score': 1.9944711},\n",
-       " {'id': '393135',\n",
-       "  'title': 'Harry Potter and the Ten Years Later',\n",
-       "  'score': 1.929232}]"
-      ]
-     },
-     "execution_count": 221,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Search request:\n",
+      "{\n",
+      "  \"fields\": [\n",
+      "    \"title\",\n",
+      "    \"id\",\n",
+      "    \"score\"\n",
+      "  ],\n",
+      "  \"limit\": 5,\n",
+      "  \"params\": {\n",
+      "    \"q\": \"{!ltr reRankDocs=60000 model=movie_model efi.keywords=\\\"harry potter\\\"}\"\n",
+      "  }\n",
+      "}\n",
+      "\n",
+      "Returned Documents:\n",
+      "[{'id': '570724', 'title': 'The Story of Harry Potter', 'score': 2.4261162}, {'id': '116972', 'title': 'Discovering the Real World of Harry Potter', 'score': 2.2478464}, {'id': '672', 'title': 'Harry Potter and the Chamber of Secrets', 'score': 2.0174994}, {'id': '671', 'title': \"Harry Potter and the Philosopher's Stone\", 'score': 1.9944708}, {'id': '393135', 'title': 'Harry Potter and the Ten Years Later', 'score': 1.9292316}]\n"
```

Comment:
Not sure if it matters, but seems we lost of indentation of the response, so it's a bit less readable now.